src/Tools/GenerateSdkPackages/Microsoft.VisualStudio.Debugger.Engine.nuspec

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```

Comment:
what are those deleted nuget packages?

Reply:
@genlu These were packages we were creating of Visual Studio assemblies we didn't own for our own reference purposes. The debugger now creates actual packages we use, so we don't need these anymore.